js/clinicalTimeline.js

```diff
@@ -1276,38 +1076,28 @@ var clinicalTimeline = (function(){
     return timeline;
   };
 
-  timeline.toggleTooltipOnVerticalLine = function() {
-    if (timelineBool.tooltipOnVerticalLine) {
-      timelineBool.tooltipOnVerticalLine = false;
-      $('#tooltip-controller a').text("Show tooltips on vertical-line");
-    } else {
-      timelineBool.tooltipOnVerticalLine = true;
-      $('#tooltip-controller a').text("Hide tooltips on vertical-line");
-    }
-    timeline();
+  timeline.setZoomFactor = function (zFactor) {
+    zoomFactor = zFactor;
+    clinicalTimelineVar.zoomFactor = zFactor;
   }
-  
-  timeline.toggleVerticalLineAndTrim = function(radioSelection) {
-    if (radioSelection.value === "trim") {
-      timelineBool.enableVerticalLine = false;
-      timelineBool.enableTrimmedTimeline = true;
-      $("#tooltip-controller").css("display", "none");
-    } else {
-      timelineBool.enableVerticalLine = true;
-      timelineBool.enableTrimmedTimeline = false;
-      $("#tooltip-controller").css("display", "inline-block");
-    }
-    timeline();
+
+  timeline.setOverviewX = function (x) {
+    overviewX = x;
+    clinicalTimelineVar.overviewX = x;
+  }
+
+  timeline.setTranslateX = function (x){
+    translateX = x;
   }
 
   /* start-test-code-not-included-in-build */
     //functions to be tested come here
     timeline.__tests__ = {}
     timeline.__tests__.getTrack = getTrack,
-    timeline.__tests__.daysToTimeObject = daysToTimeObject,
-    timeline.__tests__.formatTime = formatTime,
-    timeline.__tests__.roundDown = roundDown,
-    timeline.__tests__.roundUp = roundUp;
+    timeline.__tests__.daysToTimeObject = timeline.daysToTimeObject,
+    timeline.__tests__.formatTime = timeline.formatTime,
+    timeline.__tests__.roundDown = timeline.roundDown,
+    timeline.__tests__.roundUp = timeline.roundUp;
```

Comment:
SInce these are all public functions now, you don't have to expose them anymore in `__tests__`. You should change the tests to call the functions directly on the timeline object.